go.mod

```diff
@@ -3,14 +3,19 @@ module kubevirt.io/ssp-operator
 go 1.13
 
 require (
+	github.com/blang/semver v3.5.1+incompatible
```

Comment:
Why not use a compatible version? For example the newest one `v4.0.0`, or if that is too new, maybe: `v3.8.0`?

Reply:
Because according to https://pkg.go.dev/github.com/blang/semver?tab=versions
the latest v1 version is `v3.5.1+incompatible` and v4 is not compatible with `github.com/operator-framework/api`

Reply:
Ok